eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -67,6 +67,7 @@ case class RouteRequest(source: PublicKey,
                         ignoreChannels: Set[ChannelDesc] = Set.empty,
                         routeParams: Option[RouteParams] = None)
 
+case class PartialRouteRequest(hops:Seq[PublicKey])
```

Comment:
The name is misleading, it makes it seem like we have a partial route and we need the rest. How about (credits @sstone):

```suggestion
case class FinalizeRoute(hops:Seq[PublicKey])
```

Reply:
I came up with the name because the route here is not complete, I like the suggestion though!

Reply:
Addressed in 44acb29